test/Elastic.Apm.Tests/HelpersTests/TimeUtilsTests.cs

```diff
@@ -23,8 +23,10 @@ public class TimeUtilsTests
 			{ 5 * 24 * 60 * 60 * 1_000_000L, new DateTime(1970, 1, 6, 0, 0, 0, DateTimeKind.Utc) },
 		};
 
+#if NETCOREAPP2_1
```

Comment:
There is no `DateTime.UnixEpoch` on `net461`, I think there is no alternative for that - I'd say the easiest is to just not compile this part to `net461`, so I solved this one with just `#if NETCOREAPP2_1`.

Reply:
I think `#if NETCOREAPP` might be better in case we change the target to netcoreapp2.2 or later.